velox/core/QueryConfig.h

```diff
@@ -354,6 +354,15 @@ class QueryConfig {
   static constexpr const char* kDriverCpuTimeSliceLimitMs =
       "driver_cpu_time_slice_limit_ms";
 
+  /// The max bytes number of normalized keys for each row in the prefixsort.
+  /// Set to 0 (default value) to disable prefixsort.
+  static constexpr const char* kPrefixSortMaxNormalizedKeySize =
```

Comment:
Please, add these to documentation https://facebookincubator.github.io/velox/configs.html